### ISSUE TITLE
[spline-reco] Segmentation fault in spline reco (Trac #1361)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1353]
Segmentation fault shows up when I attempt to run spline-reco module, in the way that is shown in the attachment. I checked trunk and with the last release of icerec, and the same result.

Migrated from https://code.icecube.wisc.edu/ticket/1361
```json
{
    "status": "closed", 
    "changetime": "2015-09-24T09:38:45", 
    "description": "Segmentation fault shows up when I attempt to run spline-reco module, in the way that is shown in the attachment. I checked trunk and with the last release of icerec, and the same result.", 
    "reporter": "gmaggi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1443087525552045", 
    "component": "combo reconstruction", 
    "summary": "[spline-reco] Segmentation fault in spline reco", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-09-22T06:52:40", 
    "milestone": "", 
    "owner": "mvoge", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150922T06:54:07: 
* gmaggi commented:

> run in madison. segmentation fault will appear after few frames.


* gmaggi changed attachment from "" to "splinesegfault.py"

**Comment by icecube-trac** [comment]
Trac update at 20150924T09:38:45: 
* lraedel commented:

> In changeset 137914:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137914"
> pulseseries can be empty which was not checked
> fixes #1353
> ```


* lraedel changed status from "new" to "closed"
* lraedel changed resolution from "" to "fixed"
